example/lib/pages/polygon.dart

```diff
@@ -437,6 +437,40 @@ class _PolygonPageState extends State<PolygonPage> {
                   ),
                 ],
               ),
+              IgnorePointer(
+                child: ValueListenableBuilder(
+                  valueListenable: _hitNotifier,
+                  builder: (context, value, _) => MarkerLayer(
```

Comment:
A comment would be good why an IgnorePointer is used here.

Reply:
Done!